corehq/apps/domain/forms.py

```diff
@@ -420,6 +421,19 @@ class DomainGlobalSettingsForm(forms.Form):
         help_text=gettext_lazy("Name of the project to be used in SMS sent for account confirmation to users.")
     )
 
+    release_mode_visibility = BooleanField(
+        label=gettext_lazy("Enable Release Mode"),
+        required=False,
+        help_text=gettext_lazy(
+            """
+            Check this box to enable release mode setting on the app release page.
+            Enabled setting restricts user to directly mark a version 'released'
+            and allows users to do so only when they are in 'Release Mode' on the
+            release page of applications.
+            """
```

Comment:
We should also add that,

Once enabled, it cannot be changed/disabled. 
(whatever the wording seems better)

Reply:
"Once this setting is enabled, it cannot be disabled" - sounds good?

Reply:
works for me